config.yml

```diff
@@ -5,7 +5,6 @@ pipeline:
 - name: LexicalSyntacticFeaturizer
 - name: CountVectorsFeaturizer
   OOV_token: oov
-  token_pattern: (?u)\b\w+\b
```

Comment:
Why do we need this change?

Reply:
@hsm207 ,
See [this discussion](https://rasa-hq.slack.com/archives/C4Q4X95M4/p1619090669047600?thread_ts=1619089685.047300&cid=C4Q4X95M4):

It's not a big issue at all, only reasons it should to be removed are to avoid confusing users and because it produces a warning during parameter validation

This is actually the pattern applied when the analyzer parameter is set to word, which is the default:
```
token_pattern=r"(?u)\b\w+\b"
if parameters["analyzer"] == "word"
```

 In short, you can just remove it and the behaviour should stay the same.

Reply:
thanks for explaining!